src/scripts/styles/pages/_home.scss

```diff
@@ -1,3 +1,71 @@
+// home
+.popup {
+  position: fixed;
+  height: 100vh;
+  width: 100vw;
+  background-color: rgba(255, 255, 255, 0.9);
+  display: none;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 2%;
+
+  &-close {
+    position: absolute;
+    top: 2%;
+    right: 2%;
+    height: 50px;
+    width: 50px;
+
+    img {
+      width: 100%;
+      height: 100%;
+    }
+
+    &:hover {
+      cursor: pointer;
+    }
+  }
+
+  &-img {
+    height: 300px;
+    width: 300px;
+    border: 2px solid black;
+    border-radius: 5%;
+    padding: 10px;
+
+    img {
+      width: 100%;
+      height: 100%;
+    }
+  }
+
+  h2 {
+    font-size: 3rem;
+  }
```

Comment:
[OPTIONAL] To add a bit of fancy style, why don't we add the property ```text-transform: capitalize```? That way the first letter of the Pokemon's name would be uppercase 😸

Reply:
Done!